Assets/Scripts/EntityController.cs

```diff
@@ -13,6 +13,12 @@ public class EntityController : MonoBehaviour
     [Header("Movement related")]
     [Tooltip("Sets whether to use MovePosition (old) or AddForce (new). Check the code's comments to know the relevant variables")]
     public bool useOldMovement = false;
+    
+    [Range(1, 1000)]
```

Comment:
Could you either move these variables up or the header + `useOldMovement` variables down? You are essentially declaring non-movement related variables inside the movement related block.

Also, I don't know if there's a good reason for this to be have the Range attribute. Especially considering I've detected a bug with the attribute, where changing the value in the editor only works if you move the slider, not if you input it in the input box to the right of it.

Reply:
You're right, I will change it to an int only and move it to another part of the variable

Reply:
I agree with helio, just for a better organziation of the code.